modules/homa-lite/src/lib.rs

```diff
@@ -224,6 +232,29 @@ pub mod module {
 			Ok(())
 		}
 
+		/// Adjusts the total_staking_currency by the given difference.
+		/// Requires `T::GovernanceOrigin`
+		///
+		/// Parameters:
+		/// - `adjustment`: The difference in amount the total_staking_currency should be adjusted
+		///   by.
+		#[pallet::weight(< T as Config >::WeightInfo::adjust_total_staking_currency())]
+		#[transactional]
+		pub fn adjust_total_staking_currency(origin: OriginFor<T>, staking_total: BalanceAdjustment) -> DispatchResult {
```

Comment:
we already have `Amount` type, which is signed version of `Balance`

https://github.com/AcalaNetwork/Acala/blob/1915058c55bf76661cbd6af9ac367abc0099dccf/modules/currencies/src/lib.rs#L174